docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -113,17 +115,33 @@ Usually, the protocol adds the proposed block before reaching `requesttimeoutsec
 then starts, resetting the block time and round timeout timers. When `blockperiodseconds`
 expires, the protocol proposes the next new block.
 
-The time from proposing a block to adding the block is small (approximately one second) even in
-networks with geographically dispersed validators. Setting `blockperiodseconds` to your desired
-block time and `requesttimeoutseconds` to two times `blockperiodseconds` usually results in adding
-blocks every `blockperiodseconds`.
+Once `blockperiodseconds` is over, the time from proposing a block to adding the block is
+small (usually around one second) even in networks with geographically dispersed validators.
 
-!!! example
+Tune the `blockperiodseconds` and `requesttimeoutseconds` to fit your network and needs:
+
+- Set `blockperiodseconds` to your desired block time, the waiting duration between blocks.
+- Set `requesttimeoutseconds` depending on your network and the number of IBFT2 validator nodes.
+    This value requires tuning for your specific environment, but you can calculate a decent initial value
+    to obtain blocks every `blockperiodseconds` with the following formula:
 
-    An internal PegaSys IBFT 2.0 testnet has four geographically dispersed validators in Sweden,
-    Sydney, and two in North Virginia. With a `blockperiodseconds`of 5 and a
-    `requesttimeoutseconds` of 10 (`blockperiodseconds` x 2), the testnet consistently creates
-    blocks with a five second blocktime.
+    `requesttimeoutseconds = blockperiodseconds + (validators count * network delay)`
```

Comment:
I'd personally avoid making any hard-fast statements of how to calculate it - not sure on our position regarding making deployment style.
Can we say  "reduce this number until you starting having "round changes" then increase until round change is avoided?

Reply:
Done